config/elasticquent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+return array(
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Elasticsearch Client Configuration
+    |--------------------------------------------------------------------------
+    |
+    | This array will be passed to the Elasticsearch client.
+    | See configuration options here:
+    |
+    | http://www.elasticsearch.org/guide/en/elasticsearch/client/php-api/current/_configuration.html
+    */
+
+    'config' => [
+        'hosts'     => ['localhost:9200'],
```

Comment:
ovakve stvari bi trebalo u .env da idu (i u .env.example da znaju ljudi koji kasnije rade sta im sve treba)

to bi trebalo jer ako budes deploy-ovao aplikaciju negde, postoji sansa da nece bili localhost:9200 i onda neces moci da menjas. u sustini treba ti mogucnost da na svakom environmentu (dev, produkcija, stage...) mozes to da podesavas nezavisno